WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -598,4 +598,15 @@
       expect(competition.competitors.count).to eq 2
     end
   end
+
+  describe "#contains" do
+    let!(:delegate) { FactoryGirl.create :delegate, name: 'Pedro' }
+    let!(:search_comp) { FactoryGirl.create :competition, name: "Awesome Comp 2016", cityName: "Piracicaba", delegates: [delegate] }
+    it "searching with two words" do
+      expect(search_comp.contains?('abc aci')).to eq true
```

Comment:
That's what we were discussing offline with Pedro, I was rather up for returning things matching all parts as we do for omni search and such.